src/Sylius/Behat/Context/Setup/PaymentContext.php

```diff
@@ -112,6 +112,24 @@ public function theStoreHasAPaymentMethodDisabled(PaymentMethodInterface $paymen
     }
 
     /**
+     * @Given the store has disabled :paymentMethodName payment method
+     */
+    public function theStoreHasDisabledPaymentMethod($paymentMethodName)
```

Comment:
Did you see my comment about disabling the existing payment method in this step instead of creating a new one?